lang/zig/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           compiler_blacklist_versions 1.0
+PortGroup           github 1.0
+
+github.setup        ziglang zig 0.5.0
+github.tarball_from archive
+revision            0
+
+categories          lang
+platforms           darwin
+license             MIT
+maintainers         {@felix userspace.com.au:felix} openmaintainer
+
+description         Zig programming language
+
+long_description    Zig is a general-purpose programming language designed for \
+                    robustness, optimality, and maintainability.
+
+homepage            https://ziglang.org/
+
+depends_lib         port:llvm-9.0
```

Comment:
When using portgroups, it is best practice to use `depends_lib-append` to not overwrite any dependencies they may add.